src/mn_setup.c

```diff
@@ -2564,16 +2564,25 @@ static setup_menu_t gen_settings2[] = {
     MI_END
 };
 
+static void UpdateAmbientSounds(void)
+{
+    snd_ambient = default_snd_ambient;
+    S_EvictChannels();
+}
+
 static setup_menu_t sfx_settings1[] = {
 
     {"SFX Channels", S_THERMO, CNTR_X, M_THRM_SPC, {"snd_channels"},
-     .action = S_StopChannels},
+     .action = S_EvictChannels},
 
     {"Output Limiter", S_ONOFF, CNTR_X, M_SPC, {"snd_limiter"},
      .action = SetSoundModule},
 
     MI_GAP,
 
+    {"Ambient Sounds", S_ONOFF | S_LEVWARN, CNTR_X, M_SPC, {"snd_ambient"},
+     .action = UpdateAmbientSounds},
```

Comment:
Why do we need this option? If the WAD author included SNDINFO, then it's the intended experience, right?

Reply:
https://github.com/fabiangreffrath/woof/pull/2280/commits/952fed16f27fa5d5f938e1339c615e88899200da How about config-only, enabled by default? For really old computers or for those who don't want to hear ambient sounds (some players disable music even if that is the intended experience).

Reply:
Okay, but even GZDoom doesn't seem to have this option 😄